x-pack/test/cases_api_integration/security_and_spaces/tests/trial/cases/find_cases.ts

```diff
@@ -62,7 +59,7 @@ export default ({ getService }: FtrProviderContext): void => {
       suggestedSecUsers = await suggestUserProfiles({
         supertest: supertestWithoutAuth,
         req: {
-          name: secOnlyInfo.username!,
+          name: 'all_spaces',
```

Comment:
`securityPluginStart.userProfiles.suggest` seems to do a fuzzy match on the name field so `sec_user_all_spaces` was now returning another security user, `sec_user_read`. Changing it here guarantees the right profile is returned.